src/cpsr.rs

```diff
@@ -30,3 +30,36 @@ impl Cpsr {
         self.0 & 0x1000 != 0
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::cpsr::Cpsr;
+
+    #[test]
+    fn check_sign_flag_fail() {
+        let mut cpsr: Cpsr = Cpsr(0);
+        cpsr.set_signed();
+        assert!(cpsr.signed()); // cpsr.0 & 0x8000 != 0;
+    }
+
+    #[test]
+    fn check_sign_flag_pass() {
+        let mut cpsr: Cpsr = Cpsr(0);
+        cpsr.set_signed();
+        assert!(cpsr.0 & 0x80000000 != 0);
+    }
```

Comment:
What is the purpose of this tests?

Reply:
assert can be changed in this way:
`assert!(cspr.signed())`